readthedocs/restapi/views/model_views.py

```diff
@@ -206,3 +210,28 @@ class DomainViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return self.model.objects.api(self.request.user)
+
+
+class RemoteOrganizationViewSet(viewsets.ReadOnlyModelViewSet):
+    permission_classes = [IsOwner]
+    renderer_classes = [JSONRenderer, BrowsableAPIRenderer]
+    serializer_class = RemoteOrganizationSerializer
+    model = RemoteOrganization
+    queryset = RemoteOrganization.objects.all()
```

Comment:
These querysets aren't filtering by user at all, which means they return every project for every user.